hypothesis/lms
# Error message if an ApplicationInstance.tool_consumer_guid is reused

For #2679 

# Testing 

- Change a local AI tool_consumer_guid to trigger the error  (in canvas)
   `update application_instances set tool_consumer_instance_guid = 'another' where consumer_key = 'Hypothesisb3be0b33d0b5e4b1cf3aaf04e0e1819a';`

- Launch an assignment, for example: https://hypothesis.instructure.com/courses/125/assignments/873
- You should see an error dialog: 

![Screenshot from 2021-08-18 16-33-16](https://user-images.githubusercontent.com/1433832/129917448-f66630ca-b5c0-4349-8753-22215f4c4b6c.png)

- Same thing should happen when configuring an assignment and in any other LMS (for example in BB use `update application_instances set tool_consumer_instance_guid = 'another' where consumer_key = 'Hypothesc9deb2e461f88be4a8d5364';`) and launch
